recipes/root5/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "root5" %}
```

Comment:
Should we be embedding the version in the package name or can that be dropped?

Reply:
I was conflicted about this.

Root5 vs root6 is a bit like python2 vs python3. Root6 is a major step up, with some non-backward-compatible changes. For that reason, root5 and root6 will co-exist for some time more and will be both supported with critical bug fixes and so on. 

Some software nowadays needs Root5 and is not compatible (yet) with root6. So, we need to keep them both available in conda-forge and up to date.

Also, root5 and root6 have slightly different requirements.

I think having a root5 package and an (upcoming) root6 package makes dealing with this easier than just having one root package and having to play up and down with versions, especially for cases when there is a bug fix for v5 but not for v6.

However, if you have a different idea on how to handle this situation, I'm open to changes.

Reply:
Right. This discussion comes up periodically with different packages (e.g. Python (as you mentioned), Qt, Gtk, etc.). The same answer does not apply in all cases. That said, this is the perspective I would share.

Generally the concept of including versions in packages goes back to system level package managers (e.g. `apt-get`, `yum`, `brew`, etc.). In this case, the packaging system is designed around installing all packages in one place. Thus if one wants the option to install both Python 2 and Python 3 for instance, the packages actually have to be named differently and designed not to overlap (hence `python2` and `python3` packages). This same strategy is then applied to all dependent packages as well. Similar stories exist for all other packages where one might want the option of having multiple versions.

However `conda` already differs significantly in this regard by not mandating all packages be installed in the same place. Namely `conda` provides the environment concept. As such if one needs Python 2 and Python 3, they can easily create different environments that use one or the other. Similar stories exist with all other packages of this sort. If one needs to combine different environments together, there are options for that as well thanks to various community tools (e.g. `conda-wrappers`). Given these various options and the fact that packaging things under different versioned package names tends to be more work, it's generally easier to keep different versions under the same package name.

In general my advice would be not to hard code versions into package names. Mainly because users are free to create environments for either one as needed. It will also be less work for you and any downstream maintainers this way. That said, I think you are more familiar with Root than I. So would be good to know more about the use cases that would motivate having two versions of Root in the same environment.